contracts/src/PBHEntryPointImplV1.sol

```diff
@@ -124,22 +86,25 @@ contract PBHVerifierImplV1 is WorldIDImpl {
     /// @dev This function is explicitly not virtual as it does not make sense to override even when
     ///      upgrading. Create a separate initializer function instead.
     ///
-    /// @param _worldId The World ID instance that will be used for verifying proofs. If set to the
+    /// @param __worldId The World ID instance that will be used for verifying proofs. If set to the
     ///        0 addess, then it will be assumed that verification will take place off chain.
     /// @param _numPbhPerMonth The number of allowed PBH transactions per month.
     ///
     /// @custom:reverts string If called more than once at the same initialisation number.
-    function initialize(IWorldIDGroups _worldId, uint8 _numPbhPerMonth) public reinitializer(1) {
+    function initialize(IWorldIDGroups __worldId, IEntryPoint __entryPoint, uint8 _numPbhPerMonth)
```

Comment:
Maybe?

```suggestion
    function initialize(IWorldIDGroups worldId, IEntryPoint entryPoint, uint8 numPbhPerMonth)
```

I feel like it'd be in line with the [suggested naming convention](https://docs.soliditylang.org/en/latest/style-guide.html#local-and-state-variable-names)

Reply:
Yeah agreed, lets update to this imo.